src/index.js

```diff
@@ -0,0 +1,9 @@
+
+function submitVal() {
+	event.preventDefault();
```

Comment:
Where is event imported? Shouldn't it be like `function submitVal(event)`? Otherwise VSCode shows me this is deprecated.